assets/js/keyBindings.coffee

```diff
@@ -87,7 +87,8 @@ if module?
     ENTER_VISUAL      : ['v']
     ENTER_VISUAL_LINE : ['V']
 
-    EXPORT            : ['ctrl+s']
+    EXPORT_FILE       : ['ctrl+s'] # Really bad shortcuts, change
```

Comment:
I didn't mean to add them in insert mode.  I remember thinking you had done it while i was rebasing but there's a very high probability i was mistaken lol